packages/web-app-pdf-viewer/src/App.vue

```diff
@@ -1,88 +1,29 @@
 <template>
-  <main>
-    <app-top-bar v-if="!loading && !loadingError" :resource="resource" @close="closeApp" />
-    <loading-screen v-if="loading" />
-    <error-screen v-else-if="loadingError" />
-    <div v-else class="oc-height-1-1">
-      <object class="pdf-viewer oc-height-1-1 oc-width-1-1" :data="url" :type="objectType" />
-    </div>
-  </main>
+  <object class="pdf-viewer oc-width-1-1 oc-height-1-1" :data="url" type="application/pdf" />
```

Comment:
```suggestion
  <object class="pdf-viewer oc-width-1-1 oc-height-1-1" :data="url" :type="objectType" />
```

Reply:
Worth another commit or can be done in a follow-up PR?